### PR TITLE
Temp about us

### PR DESCRIPTION
**Pull request by Dobriyr**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/features/AboutUsPage/Founders/Founder/FounderCard/FounderCard.component.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <div className='founderCard'>
+            <div className='founderCardImageContainer' >
+                <img className='founderImage' src={founder.founderImage} />
+            </div>
+            <div className='foundersTextBlock borderRadiusSmall'>
+                <div className='founderCardTextContainer'>
+                    <h2>{founder.founderName}</h2>
+                    <p>{founder.founderRole}</p>
+                    <div className='founderCardLinks'>
+                        <a href={founder.founderFacebook}><img src={Facebook} /></a>
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/Founders/Founder/FounderCard/FounderCard.component.tsx:16]: all images should have alt for SEO

File: src/features/AboutUsPage/Founders/Founders.component.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import './Founders.styles.scss'
+import Founder from './Founder/Founder.component'
+import TeamMemberCard from '../TeamMemberCard/TeamMemberCard.component'
+import TeamMember from '@/models/team/team.model'
+import Inna from '@assets/images/about-us/Inna.png'
+import Natalia from '@assets/images/about-us/Nata.png'
+import InnaSmall from '@assets/images/about-us/InnaWithBg.png'
+import NataliaSmall from '@assets/images/about-us/NataliaWithBg.png'
+
+// тут лежить хардкод
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/Founders/Founders.component.tsx:10]: delete hardcode

File: src/features/AboutUsPage/Founders/Founders.styles.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
+@use "@assets/sass/_utils.functions.scss" as f;
+@use "@sass/variables/_variables.colors.scss" as c;
+@use "@assets/sass/mixins/_utils.mixins.scss" as mut;
+@use "@assets/sass/variables/_variables.fonts.scss" as ft;
+@use "@assets/sass/variables/_variables.sizes.scss" as s;
+
+.foundersBlock {
+    // height: 100%;
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/Founders/Founders.styles.scss:8]: remove commented code

File: src/features/AboutUsPage/Founders/Founders.styles.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        gap: f.pxToRem(60px);
+        height: fit-content;
+
+        @media screen and (max-width: s.$tablet) {
+            gap: f.pxToRem(20px) !important;
+        }
+        
+        .foundersTextBlock {
+            flex: 1;
+            display: flex;
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/Founders/Founders.styles.scss:31]: you can use mixins here like flex-center

File: src/features/AboutUsPage/TeamMemberCard/TeamMemberLinks/TeamMemberLinks.component.tsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+import twitter from '@assets/images/partners/twitter.webp';
+import youtube from '@assets/images/partners/youtube.webp';
+
+import { TeamMemberLinkCreateUpdate } from '@/models/team/team.model';
+
+const LogoType = [twitter, instagram, facebook, youtube];
+
+const TeamMemberLinks = (link: TeamMemberLinkCreateUpdate) => (
+            <a href={link.targetUrl}>
+                <img key={link.id}src={LogoType[link.logoType]} />
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/TeamMemberCard/TeamMemberLinks/TeamMemberLinks.component.tsx:12]: no space between key and src + add alt

File: src/features/AboutUsPage/TeamMembers/TeamMembers.component.scss [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+.slider-arrow {
+    &:hover {
+      fill: #000000;
+    }
+}
+
+.topSliderContainer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/TeamMembers/TeamMembers.component.scss:51]: use mixing for flex-center

File: src/features/AboutUsPage/TeamMembers/TeamMembersList/TeamMemberList.styles.scss [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    justify-self: center;
+    display: flex;
+    flex-wrap: wrap;
+    gap:20px;
+    justify-content: center;
+    margin-top: f.pxToRem(30px);
+    .teamMemberCardContainer {
+        display: flex;
+        justify-content: center;
+        //width: calc(25% - 20px);
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/TeamMembers/TeamMembersList/TeamMemberList.styles.scss:17]: remove commented code

File: src/features/AboutUsPage/TeamMembers/TeamPositionsSlider/TeamMembersSlider.component.tsx [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+const TeamMemberSlider: React.FC<TeamMemberSliderProps> = ({ team }) => {
+    const getSliderData = () => {
+        if (team.length > 0) {
+            return (
+                <Swiper
+                    slidesPerView="auto"
+                    centeredSlides
+                    spaceBetween={20}
+                    slideToClickedSlide
+                    //pagination={window.innerWidth < SCREEN_SIZES.phone}
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/TeamMembers/TeamPositionsSlider/TeamMembersSlider.component.tsx:22]: remove commented code

File: src/features/AboutUsPage/TeamMembers/TeamPositionsSlider/TeamMembersSlider.component.tsx [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+type TeamMemberSliderProps = {
+    team: TeamMember[];
+};
+
+const TeamMemberSlider: React.FC<TeamMemberSliderProps> = ({ team }) => {
+    const getSliderData = () => {
+        if (team.length > 0) {
+            return (
+                <Swiper
+                    slidesPerView="auto"
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/TeamMembers/TeamPositionsSlider/TeamMembersSlider.component.tsx:18]: it's better to decouple it to const sliderProps and then use {...sliderProps} in Swiper

File: src/features/AboutUsPage/TeamMembers/TeamPositionsSlider/TeamMembersSlider.component.tsx [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+                    {
+                    team.map((member) => (
+                        <SwiperSlide key = {member.id} >
+                            <TeamMemberCard {...member} />
+                        </SwiperSlide>
+                    ))}
+                </Swiper>
+            );
+        }
+        return <></>;
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/TeamMembers/TeamPositionsSlider/TeamMembersSlider.component.tsx:34]: Use operator ? : to follow same convention like in other components

File: src/features/AboutUsPage/TeamMembers/TeamPositionsSlider/TeamPostionSlider.component.tsx [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+        const activePositionId = activePosition.id;
+        props.setActive(activePositionId);
+    };
+
+    return (
+        <div className="topSliderContainer">
+            <LeftSliderArrow className="slider-arrow" alt="Previous" onClick={handlePreviousSlide} />
+            <div className="topSlider">
+                <Swiper
+                    slidesPerView={5}
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/TeamMembers/TeamPositionsSlider/TeamPostionSlider.component.tsx:41]: Decouple this to const sliderProps

File: src/features/AboutUsPage/Vacancies/Vacancies.component.tsx [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+        ()=> { 
+            JobApi.getActive()
+                .then(
+                    result => {
+                        setJobs(result);
+                    }
+                ) 
+                .catch(
+                    (e)=> {
+                        console.log(e);
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/Vacancies/Vacancies.component.tsx:20]: in production we don't need console log. Use it only in development. If you want to show user that there was a problem then use corresponding component

File: src/features/AboutUsPage/Vacancies/Vacancies.component.tsx [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+                    }
+                )  
+        },[]
+    );
+
+    return (
+        jobs.length ? 
+            <div className='aboutUsBlockContainer'>
+                <h1><div /><span>Вакансії</span><div /></h1>
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/Vacancies/Vacancies.component.tsx:29]: what is that ? Please find better way to adjust styles. Maybe <дів><аш1>Ваканції</аш1></дів>?

File: src/features/AboutUsPage/Vacancies/Vacancy/VacancyModal/VacancyModal.styles.scss [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+          background-color: c.$modal-hover-color;
+        }
+
+        .ant-modal-close-x {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+        }
+      }
+
```
Comment (Oleh-Kril) [inline src/features/AboutUsPage/Vacancies/Vacancy/VacancyModal/VacancyModal.styles.scss:92]: formating: remove empty line

File: src/features/AdminPage/JobsPage/JobsModal/JobsModal.component.tsx [hunk 15]
```diff
@@ -0,0 +1,180 @@
...
+import JobApi from '@/app/api/job/Job.api';
+
+interface Props {
+    open: boolean,
+    setOpen: React.Dispatch<React.SetStateAction<boolean>>,
+    currentId: number,
+}
+
+const JobsModal = ({ open, setOpen, currentId } : Props) => {
+    const maxLengths = {
```
Comment (Oleh-Kril) [inline src/features/AdminPage/JobsPage/JobsModal/JobsModal.component.tsx:23]: maybe it's better to use UPPER_CASE here for that kind of constants?

File: src/models/about-us/Founder.model.tsx [hunk 16]
```diff
@@ -0,0 +1,10 @@
+interface FounderProps
+{
+    founderName: string,
+    founderRole: string,
+    founderInstagram: string,
+    founderFacebook: string,
```
Comment (Oleh-Kril) [inline src/models/about-us/Founder.model.tsx:6]: I guess it's not only about Instagram and Facebook. There will be other social medias. I think it's better to have something like that : socialMediaLinks: {facebook: ..., instagram: .., website: ...., linkedIn: ...}

File: src/features/StreetcodePage/TimelineBlock/Timespan/Timespan.component.tsx [hunk 17]
```diff
@@ -17,7 +17,7 @@ const TimelineTimespan = () => {
         <div className="timeSpanContainer">
             <div className="timelineYearTicksContainer">
                 <TimelineSwiper
-                   
+
```
Comment (Oleh-Kril) [inline src/features/StreetcodePage/TimelineBlock/Timespan/Timespan.component.tsx:20]: remove empty line

File: src/features/AdminPage/JobsPage/JobsTable/JobsTable.component.tsx [hunk 18]
```diff
@@ -0,0 +1,173 @@
...
+                            setMappedJobsShort(
+                                mappedJobsShort.filter(
+                                    (j) => (j.id !== id),
+                                ),
+                            );
+                        },
+                    )
+                    .catch(
+                        (e) => {
+                            console.log(e);
```
Comment (Oleh-Kril) [inline src/features/AdminPage/JobsPage/JobsTable/JobsTable.component.tsx:35]: remove console log

**Review by Oleh-Kril — CHANGES_REQUESTED** [review]
Fix issues in ticket  https://github.com/ita-social-projects/StreetCode_Client/issues/754 because they are importent and then minor fixes in comments. Good Job !

**Review by MementoMorj — APPROVED** [review]

**Pull request merged**